framework/framework.go

```diff
@@ -24,8 +24,6 @@ import (
 
 // Config represents the configuration used to create a new operator framework.
 type Config struct {
-	// Dependencies.
-
```

Comment:
I removed the comments stating the things obvious for us.

Reply:
No strong feelings.